website/source/docs/auth/aws-ec2.html.md

```diff
@@ -843,9 +848,10 @@ in its identity document to match the one specified by this parameter.
       <li>
         <span class="param">bound_iam_role_arn</span>
         <span class="param-flags">optional</span>
-If set, defines a constraint on the EC2 instances to be associated with an IAM
-role ARN which has a prefix that matches the value specified by this
-parameter. Note that an exact match is also a prefix.
+	If set, defines a constraint on the authenticating EC2 instance that it
+must IAM role ARN specified by this parameter. Note that an exact match is also
```

Comment:
There comments on this: 
1) I think there is a disconnect between "must" and "IAM role ARN".
2) Two "Note" can be reduced to one.
3) The updated description here should be replicated on the field description of the actual endpoint field in the code.